examples/welcome.cr

```diff
@@ -7,11 +7,25 @@ client = Discord::Client.new(token: "Bot MjI5NDU5NjgxOTU1NjUyMzM3.Cpnz31.GQ7K9xw
 cache = Discord::Cache.new(client)
 client.cache = cache
 
+welcome_channel = 0_u64
```

Comment:
Just set this to `nil`.

Reply:
I set this to 0u64 because when its nil I get:
Error in welcome.cr:21: no overload matches 'Discord::Client#create_message' with types (UInt64 | Nil), String
Overloads are:
 - Discord::REST#create_message(channel_id : UInt64, content : String, embed : Embed | ::Nil = nil)

    client.create_message(welcome_channel, "Please welcome <@#{payload.user.id}> to #{guild.name}.")

Reply:
Because you need to reduce the type [with something like an if branch](https://carc.in/#/r/3d0t) as I [explained in my other comment](https://github.com/meew0/discordcr/pull/111#discussion_r160522796) so that `welcome_message` is only `UInt64` when you go to send the message.